packages/calendar/src/utils/stateless/events/sort-events-for-week.ts

```diff
@@ -7,7 +7,11 @@ export const sortEventsForWeekView = (
   const timeGridEvents: CalendarEventInternal[] = []
 
   for (const event of allCalendarEvents) {
-    if (event._isSingleDayTimed || event._isSingleHybridDayTimed) {
+    if (
+      event._isSingleDayTimed ||
+      event._isSingleHybridDayTimed ||
+      event._isSingleTime
```

Comment:
Are you sure this is needed? My first thought is that all `_isSingleTime` events would already be caught by this condition